examples/magic_image_arithmetic.py

```diff
@@ -0,0 +1,33 @@
+"""Basic example of using magicgui to create an Image Arithmetic GUI in napari."""
+import enum
+import numpy as np
+import napari
+
+
+# Enums are a convenient way to get a dropdown menu
+class Operation(enum.Enum):
+    """A set of valid arithmetic operations for image_arithmetic."""
+    add = np.add
+    subtract = np.subtract
+    multiply = np.multiply
+    divide = np.divide
+
+
+# Define our function.
+def image_arithmetic(layerA: napari.layers.Image, operation: Operation, layerB: napari.layers.Image) -> napari.layers.Image:
+    """Adds, subtracts, multiplies, or divides two image layers of similar shape."""
+    return operation.value(layerA.data, layerB.data)
+
+
+# We also use the additional `call_button` option
```

Comment:
This comment isn't particularly useful, mostly it seems to repeat the content in the line of code below it. It would be better if it explained what the `call_button` actually is/does (could link to docs?), or if the comment was removed.

Reply:
expanded comment